static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -353,6 +361,40 @@ class GroupDetails extends Component<Props, State> {
     GroupStore.onPopulateReleases(this.props.params.groupId, releases);
   }
 
+  async fetchReplaysCount() {
+    const {api, location, organization, params} = this.props;
+    const {groupId} = params;
+    const {slug: orgId} = organization;
+
+    this.setState({loadingReplaysCount: true});
+
+    const eventView = EventView.fromSavedQuery({
+      id: '',
+      name: `Replays in issue ${groupId}`,
+      version: 2,
+      fields: ['count()'],
+      query: `issue.id:${groupId}`,
+      projects: [],
+    });
+
+    try {
+      const [data] = await doDiscoverQuery<TableData>(
+        api,
+        `/organizations/${orgId}/eventsv2/`,
+        eventView.getEventsAPIPayload(location)
+      );
+
+      const replaysCount = data.data[0].count.toString();
```

Comment:
Does this need `toString()`?

Reply:
we need a `toString()` because the response inside the data is `ReactText`